tests/requirements.in

```diff
@@ -1,5 +1,22 @@
-# This requirements file is used for stable ansible docs builds
+# This requirements file is used for doc builds
 # It depends on specific, tested antsibull-docs and sphinx versions
 
 -c constraints.in # <-- contains known limitations
--r requirements-relaxed.in # <-- contains base set of dependencies
+
+# Base set of dependencies.
+
+sphinx
+sphinx-intl # translation utility used by docs/docsite/Makefile
+sphinx-notfound-page
+sphinx-ansible-theme
+antsibull-docs
+sphinx-copybutton
+
+# Known limitations for indirect/transitive dependencies.
+
+jinja2 >= 3.0.0 # used by hacking/build_library/build_ansible/command_plugins/generate_man.py and dump_keywords.py
+pyyaml >= 5.1 # used by ansible-core
+resolvelib >= 0.5.3, < 1.1.0 # used by ansible-core
+sphinx-rtd-theme>=2.0.0  # Fix 404 pages with new sphinx -- https://github.com/ansible/ansible-documentation/issues/678
+rstcheck < 6  # rstcheck 6.x has problem with rstcheck.core triggered by include files w/ sphinx directives https://github.com/rstcheck/rstcheck-core/issues/3
```

Comment:
Shouldn't sphinx-rtd-theme and rstcheck go into constraints.in?

Reply:
I got caught in a tangle of dependencies and wasn't sure but that makes sense now that you say it. I'll update. Cheers.